controllers/calendar/CalendarController.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace humhub\modules\smartVillage\controllers\calendar;
+
+use humhub\modules\calendar\helpers\RestDefinitions;
+use humhub\modules\content\components\ActiveQueryContent;
+use humhub\modules\content\components\ContentActiveRecord;
+use humhub\modules\content\models\ContentContainer;
+use humhub\modules\rest\definitions\ContentDefinitions;
+use humhub\modules\smartVillage\components\AuthBaseController;
+use humhub\modules\calendar\models\CalendarEntry;
+
+
+class CalendarController extends AuthBaseController
```

Comment:
do we need to extend with `AuthBaseController` if we do not authenticate?

Reply:
Hi @donni106 
Yes, because in AuthbaseController we defined success and error response.

Reply:
i see